taipy/core/config/checkers/_data_node_config_checker.py

```diff
@@ -217,25 +196,26 @@ def _check_generic_read_write_fct_and_args(self, data_node_config_id: str, data_
                     f"DataNodeConfig `{data_node_config_id}` must be populated with a Callable function.",
                 )
 
-    def _check_class_type(self, data_node_config_id: str, data_node_config: DataNodeConfig):
-        if data_node_config.storage_type in self._PROPERTIES_TYPES.keys():
-            for class_type, prop_keys in self._PROPERTIES_TYPES[data_node_config.storage_type]:
-                for prop_key in prop_keys:
-                    prop_value = data_node_config.properties.get(prop_key) if data_node_config.properties else None
-                    if prop_value and not isinstance(prop_value, class_type):
-                        self._error(
-                            prop_key,
-                            prop_value,
-                            f"`{prop_key}` of DataNodeConfig `{data_node_config_id}` must be"
-                            f" populated with a {'Callable' if class_type == Callable else class_type.__name__}.",
-                        )
-                    if class_type == Callable and callable(prop_value) and prop_value.__name__ == "<lambda>":
-                        self._error(
-                            prop_key,
-                            prop_value,
-                            f"`{prop_key}` of DataNodeConfig `{data_node_config_id}` must be"
-                            f" populated with a serializable Callable function but not a lambda.",
-                        )
+    def _check_property_types(self, data_node_config_id: str, data_node_config: DataNodeConfig):
```

Comment:
can you help propagate this change on enterprise as well? I'm quite certain it will fail on enterprise :D thanks!